nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc

```diff
@@ -10,6 +10,21 @@ The following release notes are for previous versions of the Custom Metrics Auto
 
 For the current version, see xref:../../../nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc#nodes-cma-autoscaling-custom-rn[Custom Metrics Autoscaler Operator release notes].
 
+[id="nodes-pods-autoscaling-custom-rn-2121-384_{context}"]
```

Comment:
**PEER REVIEWER** The following text is unchanged, only moved from the current release notes (nodes-cma-autoscaling-custom-rn.adoc) to this file.